ultra/ultra/scenarios/generate_scenarios.py

```diff
@@ -437,7 +437,17 @@ def generate_left_turn_missions(
         speed_m_per_s = float("".join(filter(str.isdigit, speed))) * 5.0 / 18.0
         hijacking_params = route_distributions["ego_hijacking_params"]
         zone_range = hijacking_params["zone_range"]
-        waiting_time = hijacking_params["wait_to_hijack_limit_s"]
+
+        wait_to_hijack_limit_s = hijacking_params["wait_to_hijack_limit_s"]
+        waiting_time = (
+            np.random.randint(
+                wait_to_hijack_limit_s[0],
+                wait_to_hijack_limit_s[1],
+            )
+            if type(wait_to_hijack_limit_s) is tuple
```

Comment:
Perhaps change to an `isinstance(...)` to be more inclusive?
```suggestion
            if isinstance(wait_to_hijack_limit_s, (list, tuple))
```
I noticed in `distributions.py` there are both lists and tuples for some of the values in the dictionaries.